### DISCUSSION TITLE
Define strong-named alias

### DISCUSSION THREAD

**Discussion by flobernd** [microsoft/typespec#3341] [Q&A] [answered]
Hi there,

is there a preferred way of defining strong-named aliases?

For models I think "model B is A" should work, but what should we do for scalar types?

**Comment by bterlson** [comment, marked as answer]
We don't have a general purpose way yet. We have discussed a `declare` keyword for this purpose. For scalars I think the best you can do is `extend` which should hopefully do what you want in emitters.

**Reply by flobernd** [reply]
Thanks for the quick response!

A general purpose keyword sounds like a great idea 🙂 

Thank you for mentioning the "extends" workaround. We will try to incorporate that into our custom emitter.
